mentor_classifier/mentor_classifier/arch/lr/predict.py

```diff
@@ -122,6 +133,7 @@ def __get_offtopic(self):
             return (
                 self.mentor.utterances_by_type["_OFF_TOPIC_"][i][0],
                 self.mentor.utterances_by_type["_OFF_TOPIC_"][i][1],
+                self.mentor.utterances_by_type["_OFF_TOPIC_"][i][2],
```

Comment:
is this whole bit covered by tests? I guess the basic idea is it tries to return 3...(what type of thing) from a random bucket of `OFF_TOPIC` items and then if there aren't at least 3, it returns just the label `OFF_TOPIC` twice?